### PR TITLE
Added more CSS properties

### PR DESCRIPTION
**Pull request by s0kil**
Added more CSS properties according to https://github.com/chromium/chromium/blob/master/third_party/blink/renderer/core/css/css_properties_ranking.json5

Addresses the compiler error at https://github.com/mint-lang/mint/issues/221

### CODE REVIEW

File: src/assets/css_properties [hunk 1]
```diff
@@ -1,3 +1,12 @@
+-epub-caption-side
+-epub-text-combine
+-epub-text-emphasis
+-epub-text-emphasis-color
+-epub-text-emphasis-style
+-epub-text-orientation
+-epub-text-transform
+-epub-word-break
+-epub-writing-mode
```
Comment (s0kil) [inline src/assets/css_properties]: Is this valid in Mint?

Comment (gdotdesign) [reply src/assets/css_properties]: It is, and properties starting with `-` (mostly browser prefixed) are not checked so we don't need to add these.

Comment (s0kil) [reply src/assets/css_properties]: Could we also check those? What other properties are there? Besides `-webkit-`, `-moz-`, `-ms-`, `-o-`?

Comment (gdotdesign) [reply src/assets/css_properties]: https://github.com/mint-lang/mint/blob/master/src/type_checkers/css_definition.cr#L21

Comment (gdotdesign) [reply src/assets/css_properties]: We could check those but there are a lot of them and keeping those up to date would be a challenge.

Also they are not used very much so I don't think it worth the effort.

Comment (s0kil) [reply src/assets/css_properties]: Looking at Autoprefixer, the mentioned prefixes are the only ones they use.
But likes you said, now days it is automated.

Comment (s0kil) [reply src/assets/css_properties]: Looks good to merge!

File: src/assets/css_properties [hunk 2]
```diff
@@ -552,4 +653,7 @@ wrap-flow
 wrap-inside
 wrap-through
 writing-mode
+x
+y
 z-index
+zoom
```
Comment (Sija) [inline src/assets/css_properties]: Missing newline.

Comment (s0kil) [reply src/assets/css_properties]: Is it going to break?

Comment (Sija) [reply src/assets/css_properties]: I guess not, but it's a good practice to have files with trailing newline.

Comment (gdotdesign) [reply src/assets/css_properties]: I don't think so, it's just code style: I have it setup that it always adds a new line on save if there is not one.

Comment (Sija) [reply src/assets/css_properties]: @gdotdesign It's more than that, see https://thoughtbot.com/blog/no-newline-at-end-of-file for more in-depth explanation.

Comment (s0kil) [reply src/assets/css_properties]: Dam it Unix.

**Pull request merged**